src/CoreBundle/CronJob/CronjobEnablingServiceInterface.php

```diff
@@ -26,12 +26,17 @@
     public function isCronjobExecutionEnabled(): bool;
 
     /**
-     * @throws CronjobEnableException
+     * @throws CronjobHandlingException
      */
     public function enableCronjobExecution(): void;
 
     /**
-     * @throws CronjobEnableException
+     * @throws CronjobHandlingException
      */
     public function disableCronjobExecution(): void;
+
+    /**
+     * @throws CronjobHandlingException
+     */
+    public function isOneCronjobRunning(): bool;
```

Comment:
How about `isCronjobRunning()`? We are interested in the information if at least one cronjob is running, not exactly one (in practice this is currently the same on the implementation level, but I wouldn't say that this is by concept).

And I think this method doesn't really fit in the interface. We would ask a service that is responsible for activation and deactivation if there's a cronjob currently running - doesn't really sound right.

Reply:
Ok introduced CronjobStateService.